org.kde.dolphin.json

```diff
@@ -9,20 +9,25 @@
         "--filesystem=/media",
         "--filesystem=/run/media",
         "--filesystem=home",
+        "--own-name=org.kde.dolphin",
         "--share=ipc",
         "--share=network",
         "--socket=fallback-x11",
         "--socket=pulseaudio",
         "--socket=wayland",
         "--system-talk-name=org.freedesktop.UDisks2",
         "--talk-name=org.freedesktop.Flatpak",
+        "--talk-name=org.kde.ActivityManager",
         "--talk-name=org.kde.JobViewServer",
+        "--talk-name=org.kde.KIOFuse",
```

Comment:
Has anyone tried whether KIO Fuse works in flatpak, e.g if the fuse filesystem is mounted at a location that can be accessed by flatpak apps?

Reply:
What uses fuse mounts and I'll test it.

Do you mean fuse mounts that are on the host file system are exposed in the Flatpak and vice versa or something else?

Reply:
KIOFuse should create a fuse mount point when browsing a remote share in Dolphin and opening a file from it in an application that doesn't natively support that protocol